src/Jellyfin.Database/Jellyfin.Database.Implementations/Entities/KeyframeData.cs

```diff
@@ -0,0 +1,42 @@
+#pragma warning disable CA2227 // Collection properties should be read only
+
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace Jellyfin.Database.Implementations.Entities;
+
+/// <summary>
+/// Keyframe information for a specific file.
+/// </summary>
+public class KeyframeData
+{
+    /// <summary>
+    /// Gets the id.
+    /// </summary>
+    /// <remarks>
+    /// Required.
+    /// </remarks>
+    [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
+    public int Id { get; private set; }
```

Comment:
Why have an integer id instead of a primary key based on ItemId and MediaSourceId?
Doing this would also add an index to made querying quicker

Reply:
I'll need to check how to do that in efcore, but makes totally sense